.doc_gen/metadata/glue_metadata.yaml

```diff
@@ -187,6 +197,16 @@ glue_StartCrawler:
             - description:
               snippet_tags:
                 - javascript.v3.glue.actions.StartCrawler
+    C++:
```

Comment:
Is this missing a CreateCrawler snippet tag? I see it in the code but not in the test build or the metadata.